### ISSUE TITLE
Talkback accessibility 

### ISSUE THREAD

**Issue by TothLikeBoth** [msasikanth/twine#762]
All of the buttons, links, and labels should have a clear description.
There is an accessibility scanner in the Google Play Store That might be helpful.
If there are any disappearing menus when you scroll, then it should have an option to enable persistent menus that always show. This would help those of us who are using a screen reader.
Sometimes the buttons at the top of the page to select which feed you launched. Do not work with the talkback.

I think with a little bit of work. This could be the best rss Reader for the blind and low vision community.**Describe the bug**

**To Reproduce**

**Screenshots**

**Feed Information**
Links to the feed that are having this issue

**Device Information**
 - Device: [e.g. Pixel 7]
 - OS: [e.g. Android 13]
 - App Version [e.g. 1.0.0]

**Stacktrace**
If present, add stacktrace to help debug the issue faster.
